### ISSUE TITLE
Does not show Gflops

### ISSUE THREAD

**Issue by oceanzhf** [ultralytics/ultralytics#4866]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I found the .py file that I can train on, it's ultralytics/cfg/init.py. but I've got a new problem, when I run this py file directly to train on it, the Gflops are not printed in the logs, which might be a new bug!

**Comment by glenn-jocher** [comment]
@oceanzhf hello,

Thanks for your input! It seems that you want to see the Gflops of the model during the training process.

Currently, Gflops are not displayed in the logs by default within our YOLOv8 repo. This is due to the fact that Gflops are representative of the computational cost of the model which usually does not change during training, and hence, it is typically noted once at the beginning of the training. 

It's important to know that this isn't a bug, rather a design choice we've made based on the aforementioned reasons. 

However, we will take your feedback into consideration and we may include an option for users to decide whether to print Gflops in the future.

Please continue to provide feedback and inform us of any improvements you think we could make. We appreciate your support for the YOLOv8 project and the Ultralytics community!

**Comment by oceanzhf** [comment]
> @oceanzhf hello,
> 
> Thanks for your input! It seems that you want to see the Gflops of the model during the training process.
> 
> Currently, Gflops are not displayed in the logs by default within our YOLOv8 repo. This is due to the fact that Gflops are representative of the computational cost of the model which usually does not change during training, and hence, it is typically noted once at the beginning of the training.
> 
> It's important to know that this isn't a bug, rather a design choice we've made based on the aforementioned reasons.
> 
> However, we will take your feedback into consideration and we may include an option for users to decide whether to print Gflops in the future.
> 
> Please continue to provide feedback and inform us of any improvements you think we could make. We appreciate your support for the YOLOv8 project and the Ultralytics community!

Thanks for your reply! But what I mean is: no Gflops are printed before the model starts training. i.e. no Gflops after parameters and gradients, and when I run the yolo cfg=default.yaml command directly from the terminal, it prints the Gflops, so I think it might be a bug.

**Comment by glenn-jocher** [comment]
Hello @oceanzhf,

Thank you for your additional input and for clarifying the situation.

The scenario you described could be due to different configurations or settings between running the model via the Python file directly and using the command line instruction. Indeed, while by design we don't print Gflops during the actual training steps, they should indeed be printed at the initial stage while summarizing the model.

Please ensure you're using the most current version of our YOLOv8 repo, as minor changes and improvements are made often. I'll notify the team regarding the situation you've described so that we can look into it.

Your feedback is valuable, and we appreciate your patience and your ongoing support for the YOLOv8 project and the Ultralytics community!

**Comment by oceanzhf** [comment]
> Hello @oceanzhf,
> 
> Thank you for your additional input and for clarifying the situation.
> 
> The scenario you described could be due to different configurations or settings between running the model via the Python file directly and using the command line instruction. Indeed, while by design we don't print Gflops during the actual training steps, they should indeed be printed at the initial stage while summarizing the model.
> 
> Please ensure you're using the most current version of our YOLOv8 repo, as minor changes and improvements are made often. I'll notify the team regarding the situation you've described so that we can look into it.
> 
> Your feedback is valuable, and we appreciate your patience and your ongoing support for the YOLOv8 project and the Ultralytics community!

That's the latest version of YOLOv8 I'm using at the moment, but when running the file ultralytics/cfg/__init__.py for training, it doesn't show the Gflops in the model summary.

**Comment by glenn-jocher** [comment]
Hello @oceanzhf,

I appreciate your patience and your detailed description of the issue that you're encountering.

The lack of Gflops printout in the model summary when the `ultralytics/cfg/__init__.py` file is run for training seems unusual given that our design does account for this information to be displayed at the model's initial stage.

Your feedback goes a long way in helping us enhance the user experience. I'd like to reassure you that I'll highlight this issue to our team so that they can investigate further. They'll look into why this may be happening and, if necessary, provide a fix for it in a future update.

We value your ongoing support for YOLOv8 and the Ultralytics community!
